### DISCUSSION TITLE
Implement AIC that takes into account correlated residuals

### DISCUSSION THREAD

**Discussion by mbakker7** [pastas/pastas#224] [Ideas]
Implement an AIC method that takes into account correlated residuals. Or does it make sense to compute it for the noise?

**Comment by raoulcollenteur** [comment]
Hmm, right now the AIC and BIC are computed using the residuals, maybe this should indeed be the noise when a noise model is present?

I never really used the AIC/BIC in Pastas, but this should probably be checked before somebody really uses it.

**Comment by dbrakenhoff** [comment]
AIC is calculated on noise and has been verified by comparing results to output AIC for LmFitSolve. This has been implemented since #263 .
